### PR TITLE
Ensure the database exists and change series name

### PR DESCRIPTION
**Pull request by jmccarty3**
## Pull Request Checklist

**Is this in reference to an existing issue?**
#### General
- [x] Update Changelog following the conventions laid out on [Keep A Changelog](http://keepachangelog.com/)
- [x] Update README with any necessary configuration snippets
- [ ] Binstubs are created if needed
- [x] RuboCop passes
- [x] Existing tests pass 
#### New Plugins
- [ ] Tests
- [ ] Add the plugin to the README
- [ ] Does it have a complete header as outlined [here](http://sensu-plugins.io/docs/developer_guidelines.html#coding-style)
#### Known Compatablity Issues

Ensure the database exists every time. Data is preserved if it already
exists
Change series name to the check name note the client name

### CODE REVIEW

**Comment by jmccarty3** [comment]
Fixed the RuboCop issues. Squashed the commits to a single commit.

File: bin/metrics-influxdb.rb [hunk 1]
```diff
@@ -63,24 +63,23 @@ def parse_output
 
   def check_status
     data = []
-    data.push(create_point(@event['client']['name'], @event['check']['status'], @event['client']['timestamp']))
+    data.push(create_point(@event['check']['name'], @event['check']['status'], @event['client']['timestamp']))
```
Comment (analytically) [inline bin/metrics-influxdb.rb:66]: why this change? it's not documented in the changelog...

Comment (jmccarty3) [inline bin/metrics-influxdb.rb:66]: The change was due to it being a mistake during the first commit. Using the
initial commit there was no indication what check the data point was
actually related to. I didn't had it in the change log as this behavior was
the original intent.
